src/content/structured/components/breadcrumbs/code.mdx

```diff
@@ -51,3 +53,326 @@ export const snippetsDefault = [
 </ComponentPreview>
 
 <ComponentDetails component="ic-breadcrumb-group" />
+
+## Variants
+
+### Slotted links
+
+export const slottedLinks = [
+  {
+    language: "Web component",
+    snippet: `<ic-breadcrumb-group>
+  <ic-breadcrumb page-title="home">
+    <ic-link>
+      <a slot="router-item" href="#">Home</a>
+    </ic-link>
+  </ic-breadcrumb>
+  <ic-breadcrumb page-title="beverages">
+    <ic-link>
+      <a slot="router-item" href="#">Beverages</a>
+    </ic-link>
+  </ic-breadcrumb>
+  <ic-breadcrumb current="page" page-title="coffee">
+    Coffee
+  </ic-breadcrumb>
+</ic-breadcrumb-group>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcBreadcrumbGroup>
+  <IcBreadcrumb>
+    <IcLink>
+      <a slot="router-item" href="/">Home</a>
+    </IcLink>
+  </IcBreadcrumb>
+  <IcBreadcrumb>
+    <IcLink>
+      <a slot="router-item" href="/">Beverages</a>
+    </IcLink>
+  </IcBreadcrumb>
+  <IcBreadcrumb current>
+    <IcLink>
+        <a slot="router-item" href="/">Coffee</a>
+    </IcLink>
+  </IcBreadcrumb>
+</IcBreadcrumbGroup>
+`,
+  },
+];
+
+<ComponentPreview snippets={slottedLinks}>
+  <IcBreadcrumbGroup>
+    <IcBreadcrumb>
+      <IcLink>
+        <a slot="router-item" href="/">
+          Home
+        </a>
+      </IcLink>
+    </IcBreadcrumb>
+    <IcBreadcrumb>
+      <IcLink>
+        <a slot="router-item" href="/">
+          Beverages
+        </a>
+      </IcLink>
+    </IcBreadcrumb>
+    <IcBreadcrumb current>
+      <IcLink>
+        <a slot="router-item" href="/">
+          Coffee
+        </a>
+      </IcLink>
+    </IcBreadcrumb>
+  </IcBreadcrumbGroup>
+</ComponentPreview>
+
+### Back
+
+export const back = [
+  {
+    language: "Web component",
+    snippet: `<ic-breadcrumb-group back-breadcrumb-only>
+  <ic-breadcrumb page-title="Home" href="#" />
+  <ic-breadcrumb page-title="Beverages" href="#" />
+  <ic-breadcrumb current="true" page-title="Coffee" href="#" />
+</ic-breadcrumb-group>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcBreadcrumbGroup backBreadcrumbOnly>
+  <IcBreadcrumb pageTitle="Home" href="#" />
+  <IcBreadcrumb pageTitle="Beverages" href="#" />
+  <IcBreadcrumb pageTitle="Coffee" href="#" current />
+</IcBreadcrumbGroup>`,
+  },
+];
+
+<ComponentPreview snippets={back}>
+  <IcBreadcrumbGroup backBreadcrumbOnly>
+    <IcBreadcrumb pageTitle="Home" href="#" />
+    <IcBreadcrumb pageTitle="Beverages" href="#" />
+    <IcBreadcrumb pageTitle="Coffee" href="#" current />
+  </IcBreadcrumbGroup>
+</ComponentPreview>
+
+### With icons
+
+export const withIcons = [
+  {
+    language: "Web component",
+    snippet: `<ic-breadcrumb-group>
+  <ic-breadcrumb pageTitle="Home" href="#">
+    <svg
```

Comment:
The code snippets for svgs can be written like <svg slot="icon" ...></svg> so the snippets don't look as cluttered. The component preview ones can stay like this though